src/app/utils.ts

```diff
@@ -0,0 +1,40 @@
+export async function fetchPostJson<T>(url: string, body: any, headers: Record<string, string> = {}): Promise<T> {
+    let res;
+    try {
+        res = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+                "x-api-key":
+                    "sk_staging_666s34kJSUV71hV78mG9XcC4eVhMntGSNfvftj4EXgKPAqvnCDtmmrWAcb26pj1PnTPi6MCFuEYSfmT6dwYbtjdbYw8p6uho6n9zT2poeRnXk6ZDpna5sTN5b6GTVn4uypihYawQgC2rW4UcMyLebS1CcrCkgveTKLjUzEZYML5KekyHApfUjSa5rncnbfGzu9kbrHrPGZfxppphcekwi7zp",
```

Comment:
I think this should be a client side key

Reply:
That way we can apply domain whitelisting, and since this is open code, we want to make sure we're encouraging people to follow best practices.

In general be super careful with keys!

Reply:
You are right. The `wallet.fund` scope is only available for server-side keys. What would you recommend the next step would be? Allow the `wallet.fund` scope to be applied to client-side keys or something different?

Reply:
Yeah that sounds good

Reply:
Okay, next steps:
1. Create a PR with the scope being allowed on client-side keys
2. Create client -side API key with authorized origins
3. Delete the above API key
4. merge this code too

Reply:
Updated to use the client-side key. The correct scope not available on staging but will update once it is.